roles/openshift_node/tasks/openvswitch_system_container.yml

```diff
@@ -1,14 +1,11 @@
 ---
 - set_fact:
-    l_use_crio: "{{ openshift_docker_use_crio | default(false) }}"
```

Comment:
setting this value is totally redundant, so we'll just consume openshift_docker_use_crio directly.  Also, consuming this variable instead of openshift.docker.use_crio is inconsistent with how most of the other variables are consumed, so bug fixed by virtue of standardizing how we consume variables passed in from users.